frontends/verific/verific.cc

```diff
@@ -410,6 +410,20 @@ bool VerificImporter::import_netlist_instance_gates(Instance *inst, RTLIL::IdStr
 		return true;
 	}
 
+	if (inst->Type() == PRIM_DFF)
```

Comment:
This block should be repeated at line 525.
Use next to test:

`yosys -p 'verific -sv dff.sv; verific -import -d dff.vo dff; show'`


```
module dff(input CLK, input D, output reg Q);
	always @(posedge CLK)
		Q <= D;
endmodule

```